modules/nw-egress-ips-about.adoc

```diff
@@ -149,6 +149,64 @@ On Azure, the following capacity limits exist for IP address assignment:
 
 For more information, see link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits?toc=/azure/virtual-network/toc.json#networking-limits[Networking limits].
 
+[id="nw-egress-ips-multi-nic-considerations_{context}"]
```

Comment:
This feature in 4.14 is TechPreview, shall we add this here ?  cc @martinkennelly

Reply:
yeah, lets do this @stevsmit